src/TestFrameworkAdapter.php

```diff
@@ -35,10 +35,11 @@
 
 namespace Infection\AbstractTestFramework;
 
-use Infection\AbstractTestFramework\Coverage\CoverageLineData;
+use Infection\AbstractTestFramework\Coverage\TestLocation;
 
 interface TestFrameworkAdapter
 {
+    // TODO: deprecate this and provide a JUnit report file path depending on the test framework?
     public const JUNIT_FILE_NAME = 'junit.xml';
```

Comment:
I would say we can remove it. It's only used inside`infection/infection` container and can be replaced with hardcoded `junit.xml` value.

This is not a test framework specific const.

Reply:
Since the framework adapter is specifying if there's a JUnit support, doesn't it make sense to provide it as well here?

Reply:
Otherwise I guess it can be removed indeed. Do you want me to do so for the next release (I mean the one that will contain that PR) or should that be done later?

Reply:
I would vote for removing. Yes, each test framework has `hasJUnitReport()`, but the name of the JUnit report is the same for any adapter, so I think we can remove it.

> Do you want me to do so for the next release (I mean the one that will contain that PR) or should that be done later?

if you have time and passion to do it right now (well, it's a couple of lines, lol) - let's not delay it, because I would avoid (BC) changes and thus new releases in the abstract package as much as possible.

Otherwise, we can create an issue, not a big deal.

Reply:
Done :)